.github/workflows/install_test.yml

```diff
@@ -8,13 +8,16 @@ on:
 name: Install Test
 jobs:
   install_ft_complete:
-    name: ${{ matrix.os}} - ${{ matrix.python_version }} install featuretools complete
-    runs-on: ubuntu-latest
+    name: ${{ matrix.os }} - ${{ matrix.python_version }} install featuretools complete
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
         python_version: ["3.7", "3.8", "3.9", "3.10"]
+        exclude:
+          - python_version: "3.10"
+          - os: macos-latest
```

Comment:
needs to be 
```
- python_version: "3.10"
  os: macos-latest
```
if you only want macos with 3.10 excluded

Reply:
fixed